test/v1/connectors/ConnectorSpec.scala

```diff
@@ -42,6 +46,13 @@ trait ConnectorSpec extends UnitSpec with Status with MimeTypes with HeaderNames
       Some("other-deductions-api")
     )
 
+  val dummyHeaderCarrierConfig: HeaderCarrier.Config =
+    HeaderCarrier.Config(
+      Seq("^not-test-BaseUrl?$".r),
+      Seq.empty[String],
+      Some("individuals-income-received-api")
```

Comment:
Think`Some("individuals-income-received-api")` should be `Some("other-deductions-api")`

Reply:
And in that case, could either rename it to `dummyTysIfsHeaderCarrierConfig` or just delete `dummyIfsHeaderCarrierConfig ` and keep `dummyHeaderCarrierConfig` as a generic one (As it is correct for both TYS and non-TYS)